Jenkinsfile

```diff
@@ -1081,8 +1082,8 @@ pipeline {
                             rpm_test_post(env.STAGE_NAME, env.NODELIST)
                         }
                     }
-                } // stage('Test CentOS 7 RPMs')
-                stage('Test RPMs on Leap 15.4') {
+                } // stage('Test RPMs on EL 8.6')
+                stage('Test RPMs on Leap 15.6') {
```

Comment:
FWIW, this stage is really only necessary if we are supporting 15.5 and 15.6 in the same release.  The functional test stage does everything this stage does and (obviously) a lot more.  This stage is just intended to sanity test the RPM build on the other supported version.

Reply:
Reverted this change